index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module "svelte-virtual-list" {
+  import { SvelteComponentTyped } from "svelte/types/runtime";
```

Comment:
You can import this from `"svelte"`, no deep import needed

Reply:
cool.